descartes_moveit/include/descartes_moveit/seed_search.h

```diff
@@ -0,0 +1,31 @@
+#ifndef SEED_SEARCH_H
+#define SEED_SEARCH_H
+
+#include <moveit/robot_state/robot_state.h>
+
+namespace descartes_moveit
+{
+namespace seed
+{
+
+
+typedef std::pair<unsigned, unsigned> JointPair;
+typedef std::vector<JointPair> JointPairVec;
+typedef std::vector<double> JointConfig;
```

Comment:
Joint positions in other classes (i.e. TrajectoryPt) are not typedefed in order to keep thing clear.  IDE tools are not very useful for decoding typedefs, which is why we do not use them.  Exceptions include internal only typedefs (i.e. not externally used) or if there is some concern that the underlying data type may change (this was the case with the Trajectory IDs).